config/webpack/CustomVersionFilePlugin.ts

```diff
@@ -0,0 +1,25 @@
+import type * as FS from 'fs';
+import type {Compiler} from 'webpack';
+
+const fs = require('fs') as typeof FS;
```

Comment:
```suggestion
const fs = require<typeof FS>('fs');
```
In case is not possible to change to ES imports

Reply:
Changed to imports 😄